README.md

```diff
@@ -26,21 +26,32 @@ Many Windows developers have found that setting up an Ubuntu virtual machine is
 1. Set up [RBENV](https://github.com/sstephenson/rbenv#homebrew-on-mac-os-x) and Ruby 2.0
   1. Add `if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi` to `~/.profile`
   1. source `~/.profile`
-  1. rbenv install 2.0.0-p451
-  1. rbenv global 2.0.0-p451
-  1. rbenv rehash
+  1. `rbenv install 2.0.0-p451`
+  1. `rbenv global 2.0.0-p451`
+  1. `rbenv rehash`
 
 ### Ubuntu 14.04
 
 1. `sudo aptitude update`
 1. `sudo aptitude upgrade`
-1. `sudo aptitude install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev nodejs npm openjdk-7-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk ruby2.0 ruby2.0-dev`
+1. `sudo aptitude install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-7-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk ruby2.0 ruby2.0-dev`
   * **Hit enter and select default options for any configuration popups**
 1. Either setup RBENV or configure your default ruby and gem version to 2.0
   1. Option A - RBENV: ([instructions](https://github.com/sstephenson/rbenv#installation))
+    1. Install RBENV and ruby-build
+    1. `rbenv install 2.0.0-p451`
+    1. `rbenv global 2.0.0-p451`
+    1. `rbenv rehash`
   1. Option B - Symlinks:
     1. Ruby: `sudo ln -sf /usr/bin/ruby2.0 /usr/bin/ruby`
     1. Gem: `sudo ln -sf /usr/bin/gem2.0 /usr/bin/gem`
+    1. <code>sudo chown \`whoami\` /usr/bin/gem/</code>`
+1. Install Node.js 0.12.4 and npm 2.10.1
+  1. Option A - nodesource repository
+    1. `curl -sL https://deb.nodesource.com/setup | sudo bash -`
+    1. `sudo aptitude install -y nodejs`
+  1. Option B - Manual install
+    1. [Nodejs.org](https://nodejs.org/download/)
```

Comment:
:+1: 
due to recent build-system changes I think we've started requiring npm > 2.0, so this is helpful